src/features/resize-columns/js/ui-grid-column-resizer.js

```diff
@@ -524,12 +524,21 @@
                 menuButton = angular.element(cell).parent()[0].querySelectorAll('.ui-grid-column-menu-button');
               }
 
+              //text becomes bold when font weight = 600
+              //this needs to be checked here because the fakeElement doesn't contain the font weight
+              var textIsBold = parseInt(angular.element(cell).css('font-weight')) > 500; 
+
               gridUtil.fakeElement(cell, {}, function(newElm) {
                 // Make the element float since it's a div and can expand to fill its container
                 var e = angular.element(newElm);
                 e.attr('style', 'float: left');
-
                 var width = gridUtil.elementWidth(e);
```

Comment:
if we called gridUtil.elementWidth(newElem) instead would that return the correct width without doing a modifier?  I presume it should be using Jquery to calculate the width, which would be much preferred than using a random modifier.

Reply:
gridUtil.elementWidth(newElm) shows the current width of the element before any resizing

Reply:
before any resizing? Where is the resizing being done?  Is that what the float: left is doing?  Otherwise, the second parameter to fakeElement deals with copying properties over.  If we can it would be good to be able to replicate the font-weight onto the fakeElement and before it's width is measured.